ws-nextjs-app/utilities/buildAvEmbedUrl/index.test.ts

```diff
@@ -0,0 +1,43 @@
+import buildAvEmbedURL from '.';
+
+describe('buildAvEmbedURL', () => {
```

Comment:
Could we try a scenario for a service without a variant?

Reply:
Added